data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/json/serializers/network_action/NetworkActionSerializer.java

```diff
@@ -1,23 +1,11 @@
 package com.farao_community.farao.data.crac_impl.json.serializers.network_action;
 
-import com.farao_community.farao.data.crac_api.ExtensionsHandler;
 import com.farao_community.farao.data.crac_api.NetworkAction;
 import com.farao_community.farao.data.crac_impl.json.serializers.AbstractRemedialActionSerializer;
 import com.farao_community.farao.data.crac_impl.remedial_action.network_action.AbstractNetworkAction;
-import com.fasterxml.jackson.core.JsonGenerator;
-import com.fasterxml.jackson.databind.SerializerProvider;
-import com.powsybl.commons.json.JsonUtil;
-
-import java.io.IOException;
 
 /**
  * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
  */
 public class NetworkActionSerializer<E extends AbstractNetworkAction> extends AbstractRemedialActionSerializer<NetworkAction, E> {
-
-    @Override
-    public void serialize(E networkAction, JsonGenerator jsonGenerator, SerializerProvider serializerProvider) throws IOException {
-        super.serialize(networkAction, jsonGenerator, serializerProvider);
-        JsonUtil.writeExtensions(networkAction, jsonGenerator, serializerProvider, ExtensionsHandler.getExtensionsSerializers());
-    }
```

Comment:
this was removed because the extensions were written twice (once by the super method and once by this one), so the writeExtensions call was removed and there was only the super call left (so no point overriding it no more)